extension/src/ohlc.rs

```diff
@@ -476,37 +813,93 @@ mod tests {
                 );
                 assert_eq!(output, None);
             }
+
+            for (x, y, z) in &[
+                ("NULL", "NULL", "NULL"),
+                ("NULL", "1.0", "1.0"),
+                ("now()", "NULL", "1.0"),
+            ] {
+                let output = select_one!(
+                    client,
+                    format!("SELECT toolkit_experimental.candlestick({x}, {y}, {z})").as_str(),
+                    String
+                );
+                assert_eq!(output, None);
+            }
+
+            // TODO: This fails with invalid utf-8 sequence of 1 bytes from index 274
+            // client.select("SET timezone TO 'UTC'", None, None);
```

Comment:
This seems bad?

Reply:
I uncommented and sprinkled some prints.  Looks like it's `output.unwrap()`.

What do you see when you do this in psql?

Reply:
pushed a fix.

Reply:
Thanks!!